mappings/net/minecraft/entity/DirigibleWithItem.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_4981 net/minecraft/entity/DirigibleWithItem
+	METHOD method_26313 travel (Lnet/minecraft/class_1308;Lnet/minecraft/class_4980;Lnet/minecraft/class_243;)Z
+		ARG 1 entity
+	METHOD method_26314 interactMob (Lnet/minecraft/class_1308;Lnet/minecraft/class_1657;Lnet/minecraft/class_1268;Z)Z
+		ARG 1 entity
+		ARG 2 player
+		ARG 3 hand
+	METHOD method_6575 isSaddled ()Z
+	METHOD method_6576 setSaddled (Z)V
+		ARG 1 saddled
+	METHOD method_6577 consumeOnAStickItem ()Z
```

Comment:
is this just boosting? maybe we can make it more clear

Reply:
called in OnAStickItem class to check if the item should lose durability